docs/building-a-form/available-components.md

```diff
@@ -170,21 +170,83 @@ A group of options where the user can only select a single item.
 
 #### Usage guidelines
 
-In `formConfig`, define this in the data definition.
+The data for a group of radio buttons will be quite similar to the data for a select field (i.e., `string` type with an `enum` property), which means the `SelectWidget` will by default be rendered.
 
-For the code implementation, see [`RadioWidget`](https://github.com/usds/us-forms-system/blob/master/src/js/widgets/RadioWidget.jsx).
+In order to override the `SelectWidget`, you must pass `'ui:widget': 'radio'` to your `uiSchema` for that field. If you also want to specify different label text for each option, you would pass `'ui:options'` to `uiSchema`.
+
+Your config for a question where the answer is selected from a group of radio buttons might look like this:
+```
+schema: {
+  type: 'object',
+  properties: {
+    favoriteAnimal: {
+      type: 'string',
+      enum: ['dog', 'cat', 'octopus', 'sloth']
+    }
+  }
+},
+uiSchema: {
+  'ui:widget': 'radio',
+  'ui:options': {
+    labels: {
+      dog: 'Dog',
+      cat: 'Cat',
+      octopus: 'Octopus',
+      sloth: 'Sloth'
+    }
+  }
+}
+```
+
+For more information about this widget, see [`RadioWidget`](https://github.com/usds/us-forms-system/blob/master/src/js/widgets/RadioWidget.jsx).
```

Comment:
Gotcha. I wasn't sure the point of redirecting them to the widgets page because this section basically contains all the info they need. We may want to reorganize some of the content here, and create a separate file for widgets and move the functionality in this file that pertains to widgets into the new one. What do you think?

Reply:
> We may want to reorganize some of the content here, and create a separate file for widgets and move the functionality in this file that pertains to widgets into the new one. What do you think?

Yeah, I actually had a thought about combining this with what's left of the old "Creating a form config file" topic I tore up over in https://github.com/usds/us-forms-system/pull/167/files#diff-8dbd886edc52aa4c211a3838bff6dfbcR1 (see my comment in https://github.com/usds/us-forms-system/pull/167/files#r203158103).